keps/sig-scheduling/902-non-preempting-priorityclass/README.md

```diff
@@ -146,13 +146,108 @@ and are not requesting additional changes.
 * Test coverage includes non-preempting use cases.
 * Conformance requirements for non-preempting PriorityClasses are agreed upon.
 
-## Testing Plan
+### Risks and Mitigations
+
+The new feature may malfuction,
+or existing preemption functionality may be impaired.
+New tests (covering both nonpreepting workloads and mixed workloads),
+and the existing preempting PriorityClass tests should be used to prove stability.
+
+## Design Details
+### Testing Plan
 Add detailed unit and integration tests for nonpreempting workloads.
 
 Add basic e2e tests, to ensure all components are working together.
 
 Ensure existing tests (for preempting PriorityClasses) do not break.
 
+### Graduation Criteria
+#### Alpha (v1.15):
+
+- [x] Support NonPreemptingPriority in PriorityClasses
+
+#### Beta (v1.19):
+
+- [ ] Add integration test for NonPreemptingPriority.
+- [ ] Graduate NonPreemptingPriority to Beta.
+- [ ] Update documents to reflect the changes.
+
+
+
+## Production Readiness Review Questionnaire
+
+### Feature enablement and rollback
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate
+    - Feature gate name: NonPreemptingPriority
+    - Components depending on the feature gate: 
+      - kube-apiserver 
+      - kube-scheduler
+
+* **Does enabling the feature change any default behavior?**
+  No
+
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+  the enablement)?**
+  Yes, the feature can be disabled if the PreemptionPolicy isn't set.
```

Comment:
If a med priority Pod is created (scheduled) with PreemptionPolicy=never when the feature is enabled, and then the feature gets disabled, can a high priority Pod preempt the med priority Pod?

Reply:
If the feature gets disabled, the high priority Pod can preempt the the med priority Pod, but the a med priority Pod with PreemptionPolicy=never can't preempt  the other pods with lower priority Pod. @Huang-Wei

Reply:
ok.